### ISSUE TITLE
Retreiving fluorescent protein data from a public API

### ISSUE THREAD

**Issue by manulera** [manulera/genestorian_data_refinement#32]
Hi Anamika,

I think we should retrieve most of the fluorescent protein data from www.fpbase.org. They have quite an extensive list, and they also have aliases of proteins. For this, you should write a script in `get_data`, call it `get_fpbase_data.py` or something like that. This script should make a request to their graph database and get the full list of fluorescent proteins and their aliases, and convert them to a toml file in the same format as `allele_components/tags.toml`, but stored in `data/tags.toml`. We should keep an `allele_components/tags.toml` anyway, as there are some tags that are not in this database (the non-fluorescent ones). There is currently no example, but you can delete all the examples listed and add these two:

```
[tag.myc]
name = 'myc'
ref = ''

[tag.flag]
name = 'flag'
ref = ''
```

The very short documentation for the api entrypoint is here: https://help.fpbase.org/api/introduction#graphql

In this link you can test your queries interactively before writing code: https://www.fpbase.org/graphql/

I let you figure out the query for yourself, but the data conversion from API to toml should be:

| API      | toml |
| ----------- | ----------- |
| proteins.name      | name       |
| proteins.aliases  | synonyms       |
| proteins.primaryReference.doi  | reference       |

In summary:

* [ ] Write the script to save the data in `data/tags.toml`
* [ ] Add it to gitignore (it should not be committed, instead generated before the analysis calling the script)
  * [ ] Document this in #30 
* [ ] Delete redundant tags in `allele_components/tags.toml` and add the new ones
* [ ] Use both tags.toml in the analysis (at some point we will merge them, but let's leave that for now)

**Comment by manulera** [comment]
Minimal example of request:

```python
# %%

import requests

query = '''{
    proteins {
        id,
        name,
    }
}'''

endpoint = 'https://www.fpbase.org/graphql/'

r = requests.post(endpoint, json={'query': query})

print(r.content)


```

**Comment by manulera** [comment]
🚀 🚀 🚀 🚀 🚀 🚀 🚀
